src/datasources/asset-calibration/AssetCalibrationDataSource.test.ts

```diff
@@ -120,6 +120,28 @@ const monthLocationGroupCalibrationForecastResponseMock: CalibrationForecastResp
   }
 }
 
+const modelWorkspaceGroupCalibrationForecastResponseMock: CalibrationForecastResponse =
```

Comment:
You can probably add one more test (which will also be closer to the real-life usecase) of grouping only by workspace.